### PR TITLE
doc: update comments in voljournal

### PR DESCRIPTION
**Pull request by Yuggupta27**
Update spell errors and comments in
voljournal.go

Signed-off-by: Yug Gupta <yuggupta27@gmail.com>

---

<details>
<summary>Show available bot commands</summary>

These commands are normally not required, but in case of issues, leave any of
the following bot commands in an otherwise empty comment in this PR:

- `/retest ci/centos/<job-name>`: retest the `<job-name>` after unrelated
  failure (please report the failure too!)
- `/retest all`: run this in case the CentOS CI failed to start/report any test
  progress or results

</details>

### CODE REVIEW

File: internal/journal/voljournal.go [hunk 1]
```diff
@@ -514,16 +514,16 @@ held, to prevent parallel operations from modifying the state of the omaps for t
...
 	- imagePool: Pool where the image/subvolume is created
 	- imagePoolID: pool ID of the imagePool
 	- reqName: Name of the volume request received
-	- namePrefix: Prefix to use when generating the image/subvolume name (suffix is an auto-genetated UUID)
+	- namePrefix: Prefix to use when generating the image/subvolume name (suffix is an auto-generated UUID)
 	- parentName: Name of the parent image/subvolume if reservation is for a snapshot (optional)
 	- kmsConf: Name of the key management service used to encrypt the image (optional)
-        - volUUID: UUID need to be reserved instead of auto-generating one (this is
-          useful for mirroring and metro-DR)
+    - volUUID: UUID need to be reserved instead of auto-generating one (this is useful for
```
Comment (Madhu-1) [inline internal/journal/voljournal.go]: alignment problem.

File: internal/journal/voljournal.go [hunk 2]
```diff
@@ -581,6 +580,9 @@ func (conn *Connection) ReserveName(ctx context.Context,
 		nameKeyVal = volUUID
 	}
 
+	// After generating the UUID Directory omap, we populate the csiDirectory
+	// omap with a key-value entry to map the request to the backend volume:
+	/// `csiNameKeyPrefix + reqName: nameKeyVal`
```
Comment (Madhu-1) [inline internal/journal/voljournal.go]: `///` to `//` or use multiline comment?

**Review by Madhu-1 — COMMENTED** [review]
small nit

File: internal/journal/voljournal.go (hunk 2 above)
Comment (Yuggupta27) [reply internal/journal/voljournal.go]: Thanks for catching the typo! Updated :+1:

**Review by yati1998 — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**